model/event/host_event_finder.go

```diff
@@ -21,13 +22,14 @@ type hostStatusDistro struct {
 func (s *hostStatusDistro) MarshalBSON() ([]byte, error)  { return mgobson.Marshal(s) }
 func (s *hostStatusDistro) UnmarshalBSON(in []byte) error { return mgobson.Unmarshal(in, s) }
 
-func getRecentStatusesForHost(ctx context.Context, hostId string, n int) (int, []string) {
-	or := ResourceTypeKeyIs(ResourceTypeHost)
-	or[TypeKey] = EventHostTaskFinished
-	or[ResourceIdKey] = hostId
+func getRecentStatusesForHost(ctx context.Context, hostId string, hostProvisionTime time.Time, n int) (int, []string) {
```

Comment:
nit: can we rename this to `getRecentTaskFinishedStatusesForHost` or something similar since this function ignores other host event logs